### PR TITLE
Include internal rules

### LINKED ISSUE #71: Include internal rules
The content service needs to parse and serve the internal rules too, not only external ones.

### Acceptance criteria
- Both internal and external rules are parsed and served in the current endpoint

### PR DESCRIPTION
**Pull request by Bee-lee**
# Description
adds internal rules content parsing.
Was blocked by: https://github.com/RedHatInsights/insights-results-smart-proxy/pull/85

Fixes #71 

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps
`make before_commit` + testing alongside smart-proxy

### CODE REVIEW

File: update_rules_content.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# limitations under the License.
+
+# Updates the ./tests/content/ok/ directory with the latest rules to test against
+
+function clean_up() {
+    rm -rf "$CLONE_TEMP_DIR"
+}
+trap clean_up EXIT
+
+RULES_REPO="https://gitlab.cee.redhat.com/ccx/ccx-rules-ocp.git"
```
Comment (quarckster) [inline update_rules_content.sh:23]: I wouldn't expose internal urls. Please replace it by env var.

File: update_rules_content.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
+CONTENT_DIR="${SCRIPT_DIR}/rules-content"
+TUTORIAL_RULE_CONTENT_DIR="${SCRIPT_DIR}/rules/tutorial/content"
+
+CLONE_TEMP_DIR="${SCRIPT_DIR}/.tmp"
+RULES_CONTENT="${CLONE_TEMP_DIR}/content/"
+
+echo "Attempting to clone repository into ${CLONE_TEMP_DIR}"
+
+if ! git clone "${RULES_REPO}" "${CLONE_TEMP_DIR}"
```
Comment (quarckster) [inline update_rules_content.sh:33]: Please clone `latest-release` ref. See `git clone` example in Dockerfile.

File: update_rules_content.sh (hunk 1 above)
Comment (Bee-lee) [reply update_rules_content.sh]: removed

File: update_rules_content.sh (hunk 2 above)
Comment (Bee-lee) [reply update_rules_content.sh]: removed

**Pull request merged**